packages/app/android/app/build.gradle

```diff
@@ -51,15 +51,46 @@ android {
         targetSdkVersion flutter.targetSdkVersion
         versionCode flutterVersionCode.toInteger()
         versionName flutterVersionName
+        manifestPlaceholders = [applicationLabel: "Meli"]
     }
 
     buildTypes {
+        debug {
+            applicationIdSuffix ".debug"
+            debuggable true
+            manifestPlaceholders = [applicationLabel: "Meli (Debug)"]
+        }
+        profile {
+            applicationIdSuffix ".profile"
+            manifestPlaceholders = [applicationLabel: "Meli (Profile)"]
+        }
         release {
             // TODO: Add your own signing config for the release build.
             // Signing with the debug keys for now, so `flutter run --release` works.
             signingConfig signingConfigs.debug
         }
     }
+
+    flavorDimensions "appType"
+
+    productFlavors {
+        normal {
```

Comment:
not in love with this name but couldn't think of a better one. open to suggestions

Reply:
How about buildFlavours or appFlavours?

Reply:
to clarify, I'm talking about the "normal" and "type" wording here. "productFlavors" is a required field for configuring Android Gradle

Reply:
Ah, pardon. How about "production" then?

Reply:
I was considering that but then it gets kind of confusing because the naming is a matrix combination of the type of build i.e. debug, profile, and release, and these flavors.

so having something like productionDebug seemed a little confusing to me.

maybe something like "default"? or maybe "normal" is good enough 😂

Reply:
Hahaha, sorry, i think I'm still learning how this exactly works. Thank you for being patient 🙃

I like "default" then, but "normal" also works fine.

Reply:
oh actually I think i tried "default" but I think it's some reserved word in Gradle, so the build process complains about a syntax error 🙃